kindly help, my magenta insallation failed with     error: command 'gcc' failed with exit status 1

gcc -pthread -B /home/sree1/anaconda3/envs/magenta/compiler_compat -Wl,--sysroot=/ -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -D__LINUX_ALSA__ -D__UNIX_JACK__ -Isrc -I/home/sree1/anaconda3/envs/magenta/include/python2.7 -c src/RtMidi.cpp -o build/temp.linux-x86_64-2.7/src/RtMidi.o
    cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++ [enabled by default]
    src/RtMidi.cpp:1101:28: fatal error: alsa/asoundlib.h: No such file or directory
     #include <alsa/asoundlib.h>
                                ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/home/sree1/anaconda3/envs/magenta/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-A4yg5C/python-rtmidi/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-LdOW72-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-A4yg5C/python-rtmidi/

--- a/README.md
+++ b/README.md
@@ -75,6 +75,12 @@ Install the Magenta pip package:
 pip install magenta
 ```
 
+Note that in order to install the `rtmidi` package that we depend on, you may need to install headers for some sound libraries. On Linux, this command should install the necessary packages:
+
+```
+sudo apt-get install build-essential libasound2-dev libjack-dev
+```
+
 The Magenta libraries are now available for use within Python programs and
 Jupyter notebooks, and the Magenta scripts are installed in your path!
 

